Backpack/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -0,0 +1,65 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.calendar2
+
+import java.util.Locale
+import org.threeten.bp.LocalDate
+import org.threeten.bp.YearMonth
+import org.threeten.bp.format.TextStyle
+import org.threeten.bp.temporal.WeekFields
+
+data class CalendarParams(
+  val range: ClosedRange<LocalDate>,
+  val selectionMode: SelectionMode,
+  val cells: Map<LocalDate, Info> = emptyMap(),
+  val footers: List<YearMonth> = emptyList(),
+  val locale: Locale = Locale.getDefault(),
```

Comment:
what's this for? do we have cases where the calendar locale differs to the rest of the app?

Reply:
I feel like we're mixing configuration/state with other information here - I think this comes back to all the logic being in data classes, rather than having a viewmodel or similar where it would be easier to inject things like a locale or the current time (I believe `now` is not used here either - can this be removed?